projects/packages/videopress/src/client/block-editor/blocks/video/components/playback-panel/index.tsx

```diff
@@ -45,27 +34,34 @@ export default function PlaybackPanel( { attributes, setAttributes }: VideoContr
 	return (
 		<PanelBody title={ __( 'Playback', 'jetpack-videopress-pkg' ) }>
 			<ToggleControl
-				label={ renderControlLabelWithTooltip(
-					__( 'Autoplay', 'jetpack-videopress-pkg' ),
-					/* translators: Tooltip describing the "autoplay" option for the VideoPress player */
-					__( 'Start playing the video as soon as the page loads', 'jetpack-videopress-pkg' )
-				) }
+				label={ __( 'Autoplay', 'jetpack-videopress-pkg' ) }
 				onChange={ handleAttributeChange( 'autoplay' ) }
 				checked={ autoplay }
-				help={ __(
-					'Note: Autoplaying videos may cause usability issues for some visitors.',
-					'jetpack-videopress-pkg'
-				) }
+				help={
+					<>
+						<p>
```

Comment:
The help elements are nested into a `<p>` element, meaning we can't use `<p>`, `<div>`...

Reply:
You could try something like the following:

```jsx
help={
	<>
		{ __( 'Start playing the video as soon as the page loads.', 'jetpack-videopress-pkg' ) }
		{ autoplay && (
			<>
				<br />
				{ __(
					'Note: Autoplaying videos may cause usability issues for some visitors.',
					'jetpack-videopress-pkg'
				) }
			</>
		) }
	</>
}
```